index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>JavaScript Basics – Challenge</title>
+    <link href="css/style.css" rel="stylesheet"> 
+  </head>
+  <body>
+    <main></main>
+    <script src="js/story.js"></script>   
```

Comment:
There is an error running this branch. I believe simply removing `js/` will fix this issue. Please update thanks
<img width="504" alt="image" src="https://github.com/user-attachments/assets/6e9017b6-f176-4b97-9a51-87935d960568">
```suggestion
    <script src="story.js"></script>   
```